src/schema.js

```diff
@@ -19,23 +26,53 @@ const ToDoType = new GraphQLObjectType({
 const QueryType = new GraphQLObjectType({
   name: 'Query',
   fields: () => ({
-    findToDos: {
-      type: new GraphQLList(ToDoType),
+    findTodos: {
+      type: new GraphQLList(TodoType),
       args: {
         itemId: {
           type: GraphQLInt,
         },
       },
       resolve(_, args) {
-        const response = ToDo.find({ itemId: args.itemId });
+        const response = Todo.find({ itemId: args.itemId });
         return response;
       },
     },
   }),
 });
 
+const MutationAdd = {
+  type: new GraphQLList(TodoType),
```

Comment:
why?

Reply:
with out this, then i receive a error:
```Error: Mutation.add field type must be Output Type but got: undefined.```

Reply:
that's because a mutation needs `inputs` https://github.com/entria/graphql-dataloader-boilerplate/blob/master/src/mutation/LoginEmailMutation.js#L11
and `outputs`: https://github.com/entria/graphql-dataloader-boilerplate/blob/master/src/mutation/LoginEmailMutation.js#L43

Reply:
Thx, i am using the boilerplate now, and some concepts is clarify now and i am learning much more with this course https://www.howtographql.com
off: i talk with you in inglish with out google translate :)